File: BubbleSorting.java
class BubbleSort{
	
	void BubbleSortFunc(int a[]) {
		for(int i=0;i<a.length-1;i++) {
			for(int j=0;j<a.length-i-1;j++) {
				if(a[j]>a[j+1]) {
					   int temp = a[j+1];  
                       a[j+1] = a[j];  
                       a[j] = temp;  
				}
			}
		}
			}
	
void display(int a[]) {
	for(int i=0;i<a.length;i++) {
	System.out.println(a[i]);
	}
}
}

public class BubbleSorting {

	public static void main(String[] args) {
    BubbleSort BS = new BubbleSort();
    int a[]= {1,11,5,6,2,3};
    BS.BubbleSortFunc(a);
    BS.display(a);
	}

}

File: BubbleSorting.cpp
//C++ Bubble Sort
#include <iostream>
#include <vector>

using namespace std;

class Sort{
private:

vector<int> privateList;

public:


    //Constructor
    Sort::Sort(vector<int> list){
    
        //Copy vector into private member        
        for(int i = 0; i < list.size(); i ++){
            privateList.at(i) = list.at(i);
        }
    }

    //Sorting
    void Sort::bubbleSort(vector<int> &list){
        
        for(int i = 0; i < list.size(); i++){
            for (int j = (i + 1); j < list.size(); ++j) {


                if(list.at(i) > list.at(j)){
                    
                    int temp;

                    temp = list.at(i);
                    list.at(i) = list.at(j);
                    list.at(j) = temp;

                }
            }
        }
    }

    //Simple print method
    void Sort::print(vector<int> list){

        for(int i = 0; i < list.size(); i++){

            cout << list.at(i) << " ";

        }

    }


};